src/easystar.js

```diff
@@ -286,6 +290,23 @@ EasyStar.js = function() {
             instance.startY, null, STRAIGHT_COST));
 
         instances.push(instance);
+        return instance.id;
+    };
+
+    /**
+     * Cancel a path calculation.
+     *
+     * @param {Number} id The ID of the path being calculated.
+     *
+     **/
+    this.cancelPath = function(id) {
+        for (var i = 0; i < instances.length; i ++) {
+            var instance = instances[i];
```

Comment:
Also could this return `true` if it cancelled the path, and `false` if it didn't?